Source/JavaScriptCore/wasm/WasmOMGIRGenerator32_64.cpp

```diff
@@ -1097,11 +1094,12 @@ OMGIRGenerator::OMGIRGenerator(const ModuleInformation& info, OptimizingJITCalle
 
     // FIXME we don't really need to pin registers here if there's no memory. It makes wasm -> wasm thunks simpler for now. https://bugs.webkit.org/show_bug.cgi?id=166623
 
-    m_proc.pinRegister(GPRInfo::wasmBaseMemoryPointer);
     m_proc.pinRegister(GPRInfo::wasmContextInstancePointer);
+#if OMG_JSVALUE_32_64_PINNED_MEMORY_REGISTERS
```

Comment:
Are we planning to keep these?

Reply:
Ah, for the follow-up. I see

Reply:
To be clear, I would leave the PINNED_MEMORY_REGISTERS guard around until OMG is fully-functional. The other OMG_JSVALUE32_64 guards are temporary, to be removed by follow-up PRs that adjust more of `WasmOMGIRGenerator32_64.cpp`.